.env.example

```diff
@@ -70,6 +70,10 @@ NEXT_PUBLIC_ZENDESK_KEY=
 # Help Scout Config
 NEXT_PUBLIC_HELPSCOUT_KEY=
 
+# Inbox to send user feedback
+SEND_FEEDBACK_EMAIL=
```

Comment:
```suggestion
SEND_FEEDBACK_EMAIL=support@cal.com
```

Reply:
I don't think this is a good idea. We're going to receive feedback from self-hoster users not self-hostess themselves.